README.md

```diff
@@ -1,3 +1,15 @@
 # timesync-frontend-flask
 
 Front-end webapp for the TimeSync API.
+
+To install requirements, do:
+```sh
+$ virtualenv venv
+$ source venv/bin/activate
+(venv) $ pip install -r requirements.txt
+```
+
+To run tests, do:
+```
+(venv) $ nosetests
```

Comment:
`nose` isn't actually included in the `requirements.txt`, so this doesn't work. Should it be?

We might be able to set up multiple requirement files for different environment (`dev.txt` includes `nose`, `prod.txt` only includes what you need to run the code, etc), but the setup guide should work out of the box.